security/password_migration.rst

```diff
@@ -9,21 +9,20 @@ How to Migrate a Password Hash
     Password migration was introduced in Symfony 4.4.
 
 In order to protect passwords, it is recommended to store them using the latest
-hash algorithms. This means that if a better hash algorithm is supported on the
-system, the user's password should be rehashed and stored. Symfony provides this
-functionality when a user is successfully authenticated.
-
-To enable this, make sure you apply the following steps to your application:
+hash algorithms. This means that if a better hash algorithm is supported on your
+system, the user's password should be *rehashed* using the newer algorithm and
+stored. That's possible with the ``migrate_from`` option:
 
 #. `Configure a new Encoder Using "migrate_from"`_
 #. `Upgrade the Password`_
 #. Optionally, `Trigger Password Migration From a Custom Encoder`_
 
 Configure a new Encoder Using "migrate_from"
---------------------------------------------
+----------------------------------------------
 
-When configuring a new encoder, you can specify a list of legacy encoders by
-using the ``migrate_from`` option:
+When a better hashing algorithm becomes available, you should keep the existing
```

Comment:
Safer/Stronger instead better?

Reply:
I prefer to keep the generic word "better". I know it's generic, but that's why I preferred it in this case. For example, in the future we could have a new hash algorithm that it's as strong/safe as Argon2i, but consumes half of memory or requires half of storage. So, it wouldn't be safer/stronger, but better in other ways.